policies/networking/expected.rego

```diff
@@ -13,7 +13,8 @@ expected =
     "cica-development": "10.233.2.64/26",
     "hmpps-production": "10.233.2.128/26",
     "hmcts-development": "10.233.2.192/26",
-    "hmcts-production": "10.233.3.0/26"
+    "hmcts-production": "10.233.3.0/26",
```

Comment:
Actually these 2 blocks and associated tests should have been removed, we now allocate transit gateway and protected subnets from the general subnet set, we used to have to do this manually hence the need to check them, but this is no longer required.

Reply:
Those parts have been removed from the latest commit - thanks for the comment!